### ISSUE TITLE
Item selection popup closes when clicking into the search field

### ISSUE THREAD

**Issue by RustyBlade64** [KirkMcDonald/kirkmcdonald.github.io#145]
1. Open item selection
1. Click into the search bar with the mouse

**Actual Behaviour:**
The item selection popup closes immediately.

**Expected Behaviour:**
The item selection poput stays open as I have not selected anything yet.

I seem to just intuitively click into the search bar, even though I know that the click is not necessary because it already has autofocus. Also sometimes i want to select (and subsequently copy) the text I have entered into the search bar (in order to e.g. paste it for the next search again), which causes it to close as soon as I let go of the mouse button.

**Comment by mxdanger** [comment]
Yeah, I get that same problem too. It's a script that detects a click a click anywhere on the screen. It should be modified so that if an input inside the dropdown was clicked or the blank area around the dropdown was clicked, it should disappear. I tried to look into it with my redesign of the entire site but I am not skilled in JS.
